src/main/java/Duke.java

```diff
@@ -1,10 +1,38 @@
+import data.exception.DukeException;
+import data.task.TaskList;
+import parser.Parser;
+import parser.Parser.Command;
+import storage.Storage;
+import ui.Ui;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Ui ui = new Ui();
+        Storage storage = new Storage();
+        TaskList list = new TaskList(storage.getFile());
+        list.open();
+        Parser parser = new Parser();
```

Comment:
Maybe you can consider initializing all your variables in a section on top and leaving a line before running any logic?

Reply:
Yeah that would be neater, thanks!